src/components/DeleteAccount.vue

```diff
@@ -15,27 +15,36 @@ const dialog = ref(false)
 const processing = ref(false)
 const data = ref(props.data || {})
 const logo = ref(import.meta.env.BASE_URL + 'placeholder.jpg')
-const alert = alerts()
 const password = ref()
 
 async function deleteAccount () {
   const res = await store.deleteAccount(password.value)
   if (res.message) {
     processing.value = false
   } else {
-    await alert.message(t('deleteAccount.deleteAlert'))
+    useSystemMessagesStore().addSuccess({ message: t('deleteAccount.deleteAlert') })
   }
 }
 
+const show = () => {
+  dialog.value = true
+}
+
+const hide = () => {
+  dialog.value = false
+  password.value = ''
+}
+
+defineExpose({
+  dialog,
```

Comment:
Do you need to expose this boolean ref?

Also, I would call it "dialogShown" or something like that.